docs/get-started/VeloxGPU.md

```diff
@@ -5,44 +5,76 @@ nav_order: 9
 
-It requires to install the cuda 12.8.0, driver and the nvidia-container-toolkit.
+---
 
-Refers to [start_cudf.sh](https://github.com/apache/incubator-gluten/tree/main/dev/start_cudf.sh) 
-to set the ``host`` environment and start the container.
+## **2. Prerequisites**
+- **CUDA Toolkit**: 12.8.0 ([download](https://developer.nvidia.com/cuda-downloads?target_os=Linux)).
+- **NVIDIA Drivers**: Compatible with CUDA 12.8.
+- **Container Toolkit**: Install `nvidia-container-toolkit` ([guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)).
+- **System Reboot**: Required after driver installation.
+- **Environment Setup**: Use [`start_cudf.sh`](https://github.com/apache/incubator-gluten/tree/main/dev/start_cudf.sh) for host configuration .
 
-> You may need to reboot after install the GPU driver.
+---
 
-## GPU implementation
+## **3. Implementation Mechanics**
+- **Operator Conversion**:
+    - Velox PlanNodes → **GPU operators** when `spark.gluten.sql.columnar.cudf=true`.
+    - Falls back to CPU operators if GPU unsupported (triggers row/columnar data conversion) .
+- **Debugging**: Enable `spark.gluten.debug.enabled.cudf=true` for operator replacement logs.
+- **Memory**: Global [RMM](https://docs.rapids.ai/api/librmm/stable/) memory manager, cannot align with Spark memory system.
 
-Invokes [CUDF](https://docs.rapids.ai/api/cudf/stable/libcudf_docs/) API to support the Velox operators.
+---
 
-Suppose we have a velox PlanNode, convert it to the GPU operator or CPU operator depending on the
-config `spark.gluten.sql.columnar.cudf` which decides registering cudf driver adapter or not.
+## **4. Docker Deployment**
+```bash
+docker pull apache/gluten:centos-9-jdk8-cudf  # Pre-built GPU image
+docker run --name gpu_gluten_container --gpus all -it apache/gluten:centos-9-jdk8-cudf
+```
+- **Image Includes**: Native build cache, Gluten dependencies, Spark 3.4 environment.
 
-Besides, config `spark.gluten.debug.enabled` true can print the operator replacement information.
+---
 
-## Docker images
-This docker image contains Spark at env $SPARK_HOME, Gluten at /opt/gluten, take a try if you are interested on it.
-The Gluten has been built with Spark3.4.
-```
-docker pull apache/gluten:centos-9-jdk8-cudf
-docker run --name gpu_gluten_container --gpus all -itd apache/gluten:centos-9-jdk8-cudf
+## **5. Build & Deployment**
+#### **Dependencies**
+The OS, Spark version, Java version aligns with Gluten CPU.
+
+### **Compilation Commands**
+If building in the docker image, no need to set up script and build arrow.
+```bash
+./dev/buildbundle-veloxbe.sh --run_setup_script=OFF --build_arrow=OFF --enable_cudf=ON
 ```
 
-# Branch
-The [PR](https://github.com/facebookincubator/velox/pull/12735/) has not been merged to
-facebookincubator/velox, so use a fixed Gluten branch `cudf` and corresponding oap-project velox branch `cudf`.
+## **7. GPU Operator Support Status**
```

Comment:
If "----" is used to separate each section, please add it here also for consistency.

This section's number should be 6?